charts/newrelic-infrastructure/templates/controlplane/daemonset.yaml

```diff
@@ -4,8 +4,9 @@ kind: {{ .Values.controlPlane.kind }}
 metadata:
   namespace: {{ .Release.Namespace }}
   labels:
-    {{- include "common.labels" . | nindent 4 }}
-  name: {{ include "common.naming.fullname" . }}-controlplane
+    {{- include "newrelic.common.labels" . | nindent 4 }}
+    mode: {{ if not ( include "newrelic.common.privileged" . ) }}un{{ end }}privileged
```

Comment:
Should we make a teeny tiny helper for this? As we use it in several places and so.

Reply:
Better [now](https://github.com/newrelic/nri-kubernetes/pull/402/commits/78d6879bf6523dee1de549e27bb17a7d7bb1fdd5)?

Reply:
Solved on [#67f2496](https://github.com/newrelic/nri-kubernetes/pull/402/commits/67f2496a0bacfc8edb07e7200a6325fb4fc9b3ef)